src/v/kafka/server/group_router.h

```diff
@@ -137,7 +138,13 @@ class group_router final {
           });
     }
 
+    ss::future<std::vector<deletable_group_result>>
+    delete_groups(std::vector<group_id> groups);
+
 private:
+    using sharded_groups = absl::
+      flat_hash_map<ss::shard_id, std::vector<std::pair<model::ntp, group_id>>>;
```

Comment:
`ntp` is used so much in keys, that we should probably make it an integer tbh.

Reply:
https://github.com/vectorizedio/redpanda/discussions/607